cardano-cli/src/Cardano/CLI/Helpers.hs

```diff
@@ -101,6 +106,22 @@ ensureNewFile writer outFile blob = do
 ensureNewFileLBS :: FilePath -> ByteString -> ExceptT HelpersError IO ()
 ensureNewFileLBS = ensureNewFile BS.writeFile
 
+bounded :: forall a. (Bounded a, Integral a, Show a) => String -> Opt.ReadM a
+bounded t = Opt.eitherReader $ \s -> do
+  i <- Read.readEither @Integer s
+  when (i < fromIntegral (minBound @a)) $ Left $ t <> " must not be less than " <> show (minBound @a)
+  when (i > fromIntegral (maxBound @a)) $ Left $ t <> " must not greater than " <> show (maxBound @a)
+  pure (fromIntegral i)
+
+parseFilePath :: String -> String -> Opt.Parser FilePath
```

Comment:
I think this would fit better in `...EraBased.Options.Common`. All the modules requiring this function already depend on the `Common` module.